cohort-engine/src/main/java/com/ibm/cohort/engine/measure/CDMMeasureEvaluation.java

```diff
@@ -195,7 +197,36 @@ protected static void addDefineEvaluationToReport(MeasureReport report, CDMConte
 			}
 		}
 	}
-	
+
+	/**
+	 * Set the measure reference on a MeasureReport to the id of a given Measure.
+	 * Attempt to normalize the id to the format "Measure/ID" if possible.
+	 *
+	 * Examples:
+	 *   Measure/id1234 stays Measure/id1234
+	 *   Measure/id1234/_history/10 stays Measure/id1234/_history/10
+	 *   http://fhir-server-url/api/v4/Measure/id5 becomes Measure/id5
+	 *   http://fhir-server-url/api/v4/Measure/id7/_history/4 becomes Measure/id7/_history/4
+	 *
+	 * If the id does not contain "Measure/", then the report measure is set to the
+	 * full id of the measure.
+	 *
+	 * Example:
+	 *   id55432 stays id55432
+	 *
+	 * @param report MeasureReport on which to set the measure reference
+	 * @param measure Measure providing the id to normalize and set on the report
+	 */
+	protected static void setReportMeasureToMeasureId(MeasureReport report, Measure measure) {
```

Comment:
Pedantic: My brain wants to see this be a `String -> String` method and the call become something like...
```
report.setMeasure(normalizeMeasureId(measure.getId()));
```